helm/node-health-chart/Chart.yaml

```diff
@@ -0,0 +1,2 @@
+name: node-health-chart
+version: 1.0.0-[[ .SHA ]]
```

Comment:
Usually for tenant services that are going to be managed by chart-operator the helm templates have a bit of a different structure, where we don't use the `[[ .SHA ]]`, but release versions. See e.g. here:

https://github.com/giantswarm/kubernetes-nginx-ingress-controller/blob/master/helm/kubernetes-nginx-ingress-controller-chart/Chart.yaml

I see net-exporter and cert-exporter still have the old draughtsman way. And are also deployed to tenant clusters. Probably @rossf7 and @JosephSalisbury know more.

Reply:
Damnit I was too quick on the lgtm :(

For net-exporter and cert-exporter @JosephSalisbury is using the stable channel. This will work although I'm not keen that we have 2 approaches for the channel structure.